frontend/packages/signless-transactions/src/styles/global.css

```diff
@@ -0,0 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Anuphan:wght@300;400;500;600;700&display=swap');
+
```

Comment:
no need for additional font import there, it gets imported in every project's root.